src/test/scala/chiselTests/util/SizeOfSpec.scala

```diff
@@ -0,0 +1,56 @@
+package chiselTests.util
+
+import chisel3._
+import chisel3.stage.{ChiselGeneratorAnnotation, ChiselStage}
```

Comment:
any reason to not use the circt.stage one HERE?

Reply:
`circt.stage.ChiselStage` doesn't have a `--no-run-firrtl` option to enable just getting the annotations. The main problem isn't use of `chisel3.stage.ChiselStage`, but use of the SFC. (We should add `--no-run-firrtl` to the circt variant, but that's out of scope for this PR. Adam was going to add it after Chick also ran into the same issue.

Reply:
Because I want the interchange output, not the compiled output.